### PR TITLE
BO : Add a bunch of channels and fix

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Red Uno is geo-locked, removed from our channels.

### CODE REVIEW

File: streams/bo.m3u [hunk 1]
```diff
@@ -45,3 +53,11 @@ http://136.243.3.70:1935/TvUniversitaria/TvUniversitaria/playlist.m3u8
 https://solo.disfrutaenlared.com:1936/tuch/ngrp:tuch_all/playlist.m3u8
 #EXTINF:-1 tvg-id="XTOTV.bo" status="online",XTOTV (404p) [Not 24/7]
 http://www.channel.tevemi.com:1935/XtoTv/XtoTv/playlist.m3u8
+#EXTINF:-1 tvg-id="UPPTV.bo" status="online",UPP TV (480p) [Not 24/7]
+https://d2byzrnrdb259r.cloudfront.net/hls/upptv.m3u8
+#EXTINF:-1 tvg-id="UnitelGlobal.bo" status="online",Unitel Global
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7vsezg
+#EXTINF:-1 tvg-id="UnitelLaPaz.bo" status="online",Unitel La Paz
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x7vsfua
```
Comment (UltraHDR) [inline streams/bo.m3u:61]: down

File: streams/bo.m3u [hunk 2]
```diff
@@ -45,3 +53,11 @@ http://136.243.3.70:1935/TvUniversitaria/TvUniversitaria/playlist.m3u8
...
+#EXTINF:-1 tvg-id="UnitelCochabamba.bo" status="online",Unitel Cochabamba
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x7y922d
```
Comment (UltraHDR) [inline streams/bo.m3u:63]: down

File: streams/bo.m3u [hunk 3]
```diff
@@ -45,3 +53,11 @@ http://136.243.3.70:1935/TvUniversitaria/TvUniversitaria/playlist.m3u8
 https://solo.disfrutaenlared.com:1936/tuch/ngrp:tuch_all/playlist.m3u8
 #EXTINF:-1 tvg-id="XTOTV.bo" status="online",XTOTV (404p) [Not 24/7]
 http://www.channel.tevemi.com:1935/XtoTv/XtoTv/playlist.m3u8
+#EXTINF:-1 tvg-id="UPPTV.bo" status="online",UPP TV (480p) [Not 24/7]
+https://d2byzrnrdb259r.cloudfront.net/hls/upptv.m3u8
+#EXTINF:-1 tvg-id="UnitelGlobal.bo" status="online",Unitel Global
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x7vsezg
```
Comment (UltraHDR) [inline streams/bo.m3u:59]: down

File: streams/bo.m3u [hunk 4]
```diff
@@ -45,3 +53,11 @@ http://136.243.3.70:1935/TvUniversitaria/TvUniversitaria/playlist.m3u8
 https://solo.disfrutaenlared.com:1936/tuch/ngrp:tuch_all/playlist.m3u8
 #EXTINF:-1 tvg-id="XTOTV.bo" status="online",XTOTV (404p) [Not 24/7]
 http://www.channel.tevemi.com:1935/XtoTv/XtoTv/playlist.m3u8
+#EXTINF:-1 tvg-id="UPPTV.bo" status="online",UPP TV (480p) [Not 24/7]
+https://d2byzrnrdb259r.cloudfront.net/hls/upptv.m3u8
```
Comment (UltraHDR) [inline streams/bo.m3u:57]: doesn't work for me

Comment (LaneSh4d0w) [reply streams/bo.m3u:57]: it answers but gives a black screen when not airing online.

File: streams/bo.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply streams/bo.m3u:59]: not live atm

File: streams/bo.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply streams/bo.m3u:63]: not live atm.

File: streams/bo.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/bo.m3u:61]: not live atm

File: streams/bo.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply streams/bo.m3u:59]: image

File: streams/bo.m3u (hunk 4 above)
Comment (LaneSh4d0w) [reply streams/bo.m3u:57]: now it works.

Comment (LaneSh4d0w) [reply streams/bo.m3u:57]: snapshot

**Review by MapGuy11 — APPROVED** [review]

File: streams/bo.m3u (hunk 4 above)
Comment (UltraHDR) [reply streams/bo.m3u:57]: works thanks

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**